frontend/components/layout/NavBar.vue

```diff
@@ -1,8 +1,22 @@
 <script lang="ts" setup>
+import { useQuery } from "@vue/apollo-composable";
 import { BSIcon } from "cdh-vue-lib";
-import { mockVwr } from "~/components/form/procRegMockConfig";
+import useStaticFile from "~/composables/useStaticFile";
+import { graphql } from "~/generated/gql";
+import type { GetFirstSlugQuery } from "~/generated/gql/graphql";
 
-const firstProcRegSlug = mockVwr.steps[0].slug;
+// We only need to know the slug of the top-level form so we can link to it.
+const GET_FIRST_SLUG = graphql(`
+    query GetFirstSlug {
+        form {
+            id
+            slug
+        }
+    }
+`);
+
+const { result } = useQuery<GetFirstSlugQuery>(GET_FIRST_SLUG);
+const slug = computed<string | null>(() => result.value?.form?.slug ?? null);
```

Comment:
To avoid visiting top level form from the navbar:
```suggestion
const GET_FIRST_SLUG = graphql(`
    query GetFirstSlug {
        form {
            id
            subforms {
                id
                slug
            }
        }
    }
`);

const { result } = useQuery<GetFirstSlugQuery>(GET_FIRST_SLUG);
const slug = computed<string | null>(() => result.value?.form?.subforms?.[0]?.slug ?? null);
```